bc_obps/registration/api/_user_operators/current.py

```diff
@@ -2,26 +2,23 @@
 from common.permissions import authorize
 from django.http import HttpRequest
 from registration.constants import USER_OPERATOR_TAGS
-from registration.schema import OperatorFromUserOperatorOut, Message
+from registration.schema import CurrentUserOperatorOut, Message
 from service.data_access_service.user_service import UserDataAccessService
 from common.api.utils import get_current_user_guid
 from registration.api.router import router
-from registration.models import (
-    UserOperator,
-)
+from registration.models import UserOperator
 from service.error_service.custom_codes_4xx import custom_codes_4xx
 
-
 # "current" refers to the current user-operator (we look up the current user-operator via the current user)
-# this endpoint retrieves data about both the user-operator and the operator
+# We don't need a user_operator_id parameter (using "pending" instead) because we can look up the user_operator via the user data we receive in the middleware
+# This endpoint retrieves data about both the user-operator and the operator.
 @router.get(
     "/user-operators/current",
-    response={200: OperatorFromUserOperatorOut, custom_codes_4xx: Message},
+    response={200: CurrentUserOperatorOut, custom_codes_4xx: Message},
     tags=USER_OPERATOR_TAGS,
-    description="""Retrieves data about the current user-operator and their associated operator.
+    description="""Retrieves data about the pending user-operator and their associated operator.
```

Comment:
I find `pending` confusing here since we use `current` everywhere else. Maybe just delete it, or add a note that it returns the user-operator regardless of approval status